### ISSUE TITLE
Exception when loading a single machine 

### ISSUE THREAD

**Issue by SeeBasTStick** [hhu-stups/prob-issues#56]
After the start of the ProB2-UI when loading a single machine file, the following exception is thrown:

```
[ERROR] Uncaught exception on thread Thread[JavaFX Application Thread,5,main]
java.lang.IllegalArgumentException: Visible paragraphs' last index is [-1] but visibleParIndex was [0]
	at org.fxmisc.richtext.GenericStyledArea.visibleParToAllParIndex(GenericStyledArea.java:982)
	at org.fxmisc.richtext.ViewActions.firstVisibleParToAllParIndex(ViewActions.java:284)
	at org.fxmisc.richtext.LineNumberFactory.insertParagraphCheck(LineNumberFactory.java:172)
	at org.fxmisc.richtext.LineNumberFactory.lambda$new$2(LineNumberFactory.java:92)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:391)
	at javafx.graphics/com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
	at javafx.graphics/com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at javafx.graphics/com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277) 
```

	
It seems to happen on Linux (Mint) and Windows. The machine is successfully loaded and accessible anyways.

**Comment by favu100** [comment]
This bug also happens on MacOS. If we switch back to RichTextFX 10.0.5, the bug disappears. It seems that ProB2-UI sometimes either provide wrong input to RichTextFX, or it is a bug in RichTextFX

**Comment by dgelessus** [comment]
Hopefully fixed with hhu-stups/prob2_ui@f5f2a0b059a6347ecd0dcd4072590e3541f84dc1.
